lib/ApplicationFeatures/ApplicationServer.cpp

```diff
@@ -354,6 +358,27 @@ void ApplicationServer::collectOptions() {
       arangodb::options::makeDefaultFlags(arangodb::options::Flags::Uncommon,
                                           arangodb::options::Flags::Command));
 
+  _options->addOption(
+      "--server.api-call-recording",
+      "Record recent API calls for debugging purposes (default: true).",
+      new BooleanParameter(&_recordApiCalls),
```

Comment:
You can define here the default value, min and max value. At least for `_memoryPerApiRecordList`, and `_numberOfApiRecordLists` you have some min values